client/ui/client_ui.go

```diff
@@ -716,16 +716,43 @@ func openURL(url string) error {
 
 // checkPIDFile exists and return error, or write new.
 func checkPIDFile() error {
-	pidFile := path.Join(os.TempDir(), "wiretrustee-ui.pid")
+	pidFile := path.Join(os.TempDir(), "netbrid-ui.pid")
 	if piddata, err := os.ReadFile(pidFile); err == nil {
 		if pid, err := strconv.Atoi(string(piddata)); err == nil {
-			if process, err := os.FindProcess(pid); err == nil {
-				if err := process.Signal(syscall.Signal(0)); err == nil {
-					return fmt.Errorf("process already exists: %d", pid)
-				}
+			if pidExists(pid) {
+				return fmt.Errorf("process already exists: %d", pid)
 			}
 		}
 	}
 
 	return os.WriteFile(pidFile, []byte(fmt.Sprintf("%d", os.Getpid())), 0o664) //nolint:gosec
 }
+
+func pidExists(pid int) bool {
+	if runtime.GOOS != "windows" {
+		if process, err := os.FindProcess(pid); err == nil {
+			if err := process.Signal(syscall.Signal(0)); err == nil {
+				return true
+			}
+		}
+	}
+	// Prepare the command
+	cmd := exec.Command("tasklist", "/FI", fmt.Sprintf("PID eq %d", pid))
+
+	// Get the output
+	output, err := cmd.Output()
+	if err != nil {
+		return false
```

Comment:
Please `log.Tracef` the err and output here
```suggestion
        log.Tracef("Failed to list tasks: %v, output: %s", err, output)
		return false
```